Adding version numbers to ITSM docs (#2278)
OctopusDeploy/docs: src/pages/docs/approvals/servicenow/index.md

active, approved **Standard Change Template** (as found in the Standard Change Catalog) will instead 
result in deployments of the project creating a `Standard` (i.e. low-risk, pre-approved) change.

> To create an `Emergency` change you can select the Emergency Change setting on the deployment creation page. This is available from version **2024.2** on.

Reviewer: ```suggestion
From **2024.2** you can create an `Emergency` change by selecting the Emergency Change setting on the deployment creation page.
```

If you add a variable to your project named `Octopus.ServiceNow.ChangeRequest.Number`, then a CR will not be created, and instead, the supplied CR number will be used during the approval check. This variable can also be [scoped](/docs/projects/variables/#scoping-variables).

> From **2024.2** on this variable can be set under the `ServiceNow Change Request settings` section on the deployment creation page. Setting the CR number at the deployment level will override any predefined variable.

Reviewer: ```suggestion
From **2024.2** on this can be set under the `ServiceNow Change Request settings` section on the deployment creation page. Setting the CR number at the deployment level will override any predefined variable.
```

The title must match the format **exactly**, including the double-quotes.
:::

> ### Populating CR fields through Octopus (Available from 2024.2 on)

Reviewer: Can we make this similar to the below:

<img width="851" alt="image" src="https://github.com/OctopusDeploy/docs/assets/1035315/af3026f8-c625-461c-be93-cc8fe8ac878b">
